README.md

```diff
@@ -3,9 +3,9 @@
 This is the second release of the SpatialKey DM API command line tool. The initial release can be found at [here]https://github.com/SpatialKey/skdm/tree/v1
 
 # Introduction
-The [SpatialKey Data Management REST API](http://support.spatialkey.com/dmapi) allows developers to programmatically create and update CSV data or Shapefiles in SpatialKey without having to login.  With the Command Line Tool, typical users of SpatialKey can leverage this functionality without any programming knowledge.  You just need to learn a few of the basics and you will be all set to kicking off import and update jobs outside of SpatialKey.
+The [SpatialKey Data Management REST API] allows developers to programmatically create and update CSV data or Shapefiles in SpatialKey without having to login.  With the Command Line Tool, typical users of SpatialKey can leverage this functionality without any programming knowledge.  You just need to learn a few of the basics and you will be all set to kicking off import and update jobs outside of SpatialKey.
```

Comment:
A couple of things
- did you intend to leave the brackets around SpatialKey Data Management REST API?
- and in the last sentence you say "outside of SpatialKey" while earlier you say "in SpatialKey"

Reply:
@kevinkorngut @wandechris The brackets were part of markdown link syntax... with no link, they aren't needed anymore.

Reply:
@kevinkorngut it meant kicking off imports and updating jobs outside of the spatialkey app

Reply:
@rstehwien @kevinkorngut updated

Reply:
Thanks, this looks good. I have a number of changes I need to make to the example data to rework it in the near future and will kick off a review.